operational-documentation/runbook.json

```diff
@@ -0,0 +1,31 @@
+{
+	"dataItemID": "origami-registry-ui",
+	"name": "Origami Registry UI",
+	"description": "Get information about Origami components, services, and repositories",
+	"serviceTier": "Bronze",
+	"lifecycleStage": "Requirements",
+	"primaryURL": "https://origami-registry-ui.ft.com",
+	"gitRepository": "https://github.com/Financial-Times/origami-registry-ui",
+	"hostPlatform": "Heroku",
+	"primaryContact": {
+		"contact": [{
+			"dataItemID": "origamisupport"
+		}]
+	},
+	"programme": {
+		"contact": [{
+			"dataItemID": "internalproducts"
+		}]
+	},
+	"productOwner": {
+		"contact": [{
+			"dataItemID": "origamisupport"
+		}]
+	},
+	"technicalLead": {
+		"contact": [{
+			"dataItemID": "origamisupport"
+		}]
+	},
+	"monitoring": "TODO",
```

Comment:
Is this meant to link to the grafana dashboard?

Reply:
Yes, but as far as I understand I have to have this up and running before I get a url to link to @rowanmanning ?

Reply:
Yep, but that doesn't exist yet. This will get updated when it does